package/flows/Add_Episode_Flow.flow-meta.xml

```diff
@@ -0,0 +1,209 @@
+        </value>
+    </processMetadataValues>
+    <processMetadataValues>
+        <name>CanvasMode</name>
+        <value>
+            <stringValue>AUTO_LAYOUT_CANVAS</stringValue>
+        </value>
+    </processMetadataValues>
+    <processMetadataValues>
+        <name>OriginBuilderType</name>
+        <value>
+            <stringValue>LightningFlowBuilder</stringValue>
+        </value>
+    </processMetadataValues>
+    <processType>Flow</processType>
+    <recordCreates>
+        <name>Create_Episode</name>
+        <label>Create Episode</label>
+        <locationX>314</locationX>
+        <locationY>518</locationY>
+        <connector>
+            <targetReference>Finish_Season_Screen</targetReference>
+        </connector>
+        <inputAssignments>
+            <field>Description__c</field>
+            <value>
+                <elementReference>Episode_Description</elementReference>
+            </value>
+        </inputAssignments>
+        <inputAssignments>
+            <field>Episode_Name__c</field>
+            <value>
+                <elementReference>Episode_Name</elementReference>
+            </value>
+        </inputAssignments>
+        <inputAssignments>
+            <field>Season__c</field>
+            <value>
+                <elementReference>SeasonId.Id</elementReference>
+            </value>
+        </inputAssignments>
+        <object>Episode__c</object>
+        <storeOutputAutomatically>true</storeOutputAutomatically>
+    </recordCreates>
+    <recordLookups>
+        <name>Get_Episode_if_Exists</name>
+        <label>Get Episode if Exists</label>
+        <locationX>182</locationX>
+        <locationY>278</locationY>
+        <assignNullValuesIfNoRecordsFound>false</assignNullValuesIfNoRecordsFound>
+        <connector>
+            <targetReference>If_episode_already_exists</targetReference>
+        </connector>
+        <filterLogic>and</filterLogic>
+        <filters>
+            <field>Episode_Name__c</field>
+            <operator>EqualTo</operator>
+            <value>
+                <elementReference>Episode_Name</elementReference>
+            </value>
+        </filters>
+        <filters>
+            <field>Season__c</field>
+            <operator>EqualTo</operator>
+            <value>
+                <elementReference>SeasonId.Id</elementReference>
+            </value>
+        </filters>
+        <getFirstRecordOnly>true</getFirstRecordOnly>
+        <object>Episode__c</object>
+        <storeOutputAutomatically>true</storeOutputAutomatically>
+    </recordLookups>
+    <screens>
+        <name>Error_Episode</name>
+        <label>Error</label>
+        <locationX>50</locationX>
+        <locationY>518</locationY>
+        <allowBack>true</allowBack>
+        <allowFinish>true</allowFinish>
+        <allowPause>false</allowPause>
+        <fields>
+            <name>Display_error_Episode</name>
+            <fieldText>&lt;p&gt;The episode already exists!&lt;/p&gt;</fieldText>
```

Comment:
Maybe change it into: "The episode with the same name(or number) already exist"

Reply:
Done